src/OpenLoco/Vehicles/VehicleHead.cpp

```diff
@@ -2676,9 +2676,16 @@ namespace OpenLoco::Vehicles
 
                 break;
             }
+
+            bool stationHadPreviousCargo = cargoStats.quantity != 0;
             cargoStats.quantity = Math::Bound::add(cargoStats.quantity, cargo.qty);
             station->updateCargoDistribution();
-            cargoStats.enroute_age = std::max(cargoStats.enroute_age, cargo.numDays);
+            cargoStats.enrouteAge += cargo.numDays;
```

Comment:
We risk an overflow here, right?

Reply:
Oh true, I've changed it to use the bound like above, idk how I haven't noticed that before.